snapcraft/extensions/kde_neon_6.py

```diff
@@ -87,8 +93,11 @@ def is_experimental(base: Optional[str]) -> bool:
 
     @overrides
     def get_app_snippet(self) -> Dict[str, Any]:
+        command_chain = ["$SNAP/kf6/snap/command-chain/desktop-launch6"]
```

Comment:
May be we can do it in the gnome way and use a interim command chain instead of directly calling the command-chain from within the content snap

Reply:
Straight from gnome 

# GTK launcher specific part
# This is not used with the gnome extension for
# core22 and later, please see
# https://github.com/canonical/snapcraft-desktop-integration

This is correct. I followed gnome, I just failed to update path.

Reply:
I just pushed the interim command chain like gnome does. Thanks !